docs/source/library-user-guide/upgrading.md

```diff
@@ -21,9 +21,6 @@
 
 ## DataFusion `50.0.0`
 
-**Note:** DataFusion `50.0.0` has not been released yet. The information provided in this section pertains to features and changes that have already been merged to the main branch and are awaiting release in this version.
-You can see the current [status of the `50.0.0 `release here](https://github.com/apache/datafusion/issues/16799)
```

Comment:
Do we need to add a `51.0.0` section now but with this message updated?

Reply:
Good question, I'm not quite sure, since I think `51.0.0` would be empty for now. cc: @alamb

Reply:
Based off this I think we add the `51.0.0` section: https://github.com/apache/datafusion/commit/8d74be8ce4a4dad4fbc4f1efa5a20db5a47034aa